lib/libc/string/memcpy.c

```diff
@@ -1,5 +1,5 @@
 #include <sys/cdefs.h>
 __FBSDID("$FreeBSD$");
 
-#define	MEMCOPY
-#include "bcopy.c"
+#define	MEMCPY
+#include "../mips/string/jdw57_memcpy.c"
```

Comment:
These changes should be reverted or wrapped in `#ifdef __mips__`.  Probably reverted and replaced with a memcpy.c (etc) in lib/libc/mips/string.  Alternatively, you could fix jdw57_memcpy.c to be platform independent.

Reply:
I hope I've fixed this one correctly. I've just reverted as best I could to what it was before.

Reply:
there should be a tab rather than a space after then define to remove the whole diff.  memcpy_c.c, memmove.c, and memmove_c.c need similar treatment.